notebooks/03_raster_zonal_stats.ipynb

```diff
@@ -354,8 +354,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # Single band
</pre><p>This cell and the next one seem to be both effectively single band with multiple aggregations. You could</p><ol><li>Remove this cell</li><li>Call this as <code>Single band with single aggregation</code> and then only use one <code>func</code>  </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>